STL_Engine/Convert/ToPolyline.cs

```diff
@@ -81,6 +81,26 @@ public static List<Polyline> ToPolyline(this PolySurface psurface)
             return psurface.IExternalEdges().Select(x => x.ICollapseToPolyline(BH.oM.Geometry.Tolerance.Angle)).ToList();
         }
 
+        public static List<Polyline> ToPolyline(this Mesh mesh)
+        {
+            List<Polyline> polylines = new List<Polyline>();
+            foreach (Face face in mesh.Faces)
+            {
+                List<Point> controlPoints = new List<Point>();
+                controlPoints.Add(mesh.Vertices[face.A]);
+                controlPoints.Add(mesh.Vertices[face.B]);
+                controlPoints.Add(mesh.Vertices[face.C]);
+                if (face.D != -1)
+                {
+                    controlPoints.Add(mesh.Vertices[face.D]);
+                }
+                Polyline polyline = new Polyline() { ControlPoints = controlPoints };
```

Comment:
This won't produce a closed polyline, is that what's after? If we want a closed polyline we need to do:
`controlPoints.Add(controlPoints.First());`
just above this line.

Reply:
Thanks @FraserGreenroyd - tested and this eliminates the warning that @BingWangUS Bing mentions in the PR 👍